### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by FinnKr**
Added Dockerfile (and .dockerignore).
Added a Docker section to the README.

This is related to issue   #8.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# pull official base image
+FROM python:3.8.6
+
+RUN apt-get install libsndfile1
+
+RUN pip install --no-cache datasets transformers soundfile numpy torch torchaudio
+
+COPY . .
```
Comment (wannaphong) [inline Dockerfile:8]: Can you add `RUN pip install -e .` after this line?

Comment (FinnKr) [reply Dockerfile:8]: Oh yeah. Sure thin. I also removed the README from the .dockerignore as it is embedded in the setup.

Comment (wannaphong) [reply Dockerfile:9]: > Oh yeah. Sure thing. I also removed the README from the .dockerignore as it is embedded in the setup.

It is before `COPY . .`. I think it should run `RUN pip install -e .` after `COPY . .`.

Comment (FinnKr) [reply Dockerfile:8]: Oh I'm so sorry. Must be getting dizzy 🥴. I just tried it in the already existing test container on my machine and forgot that the relevant files sure need to be copied.😵‍💫 Should be fixed now.

**Review by wannaphong — APPROVED** [review]
Thank you! Have a good day! :smile:

**Pull request merged**